format/spec.md

```diff
@@ -313,123 +313,81 @@ The set of metadata columns is:
 
 ### Row Lineage
 
-In Specification V3, an Iceberg Table can declare that engines must track row-lineage of all newly created rows. This
-requirement is controlled by setting the field `row-lineage` to true in the table's metadata. When true, two additional 
-fields in data files will be available for all rows added to the table.
+In v3 and later, an Iceberg table can track row lineage fields for all newly created rows.  Row lineage is enabled by setting the field `row-lineage` to true in the table's metadata. When enabled, engines must maintain the `next-row-id` table field and the following row-level fields when writing data files:
 
-* `_row_id` a unique long for every row. Computed via inheritance for rows in their original datafiles 
-and explicitly written when the row is moved to a new file.
-* `_last_update` the sequence number of the commit which last updated this row. The value is computed via inheritance for 
-rows in their original file or in files where the row was modified.
+* `_row_id` a unique long for every row. Assigned via inheritance when a row is added and the existing value is explicitly written when the row is written to a new file.
```

Comment:
I still think "added" here is ambigous, which is why I had newly created. Perhaps "first added to the table".

I think the other half of the sentence is also a bit confusing grammatically to me. I'll try to do a little rewording here.